catalog/spatial/registry/registry-rest-endpoint/src/main/java/org/codice/ddf/registry/rest/endpoint/HandlebarsHelper.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.registry.rest.endpoint;
+
+import java.io.IOException;
+
+import com.github.jknack.handlebars.Options;
+
+public class HandlebarsHelper {
+
+    public CharSequence ifeq(String value, Options options) throws IOException {
```

Comment:
The jacoco report showed significant drops because the if statements were not tested when it was contained with the handlebars.  Creating a separate class allowed to check if the helper through `HandlebarsHelperTest.java`. Also, this allows for more handlebar helpers to be added if needed.